docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM openjdk:8u181-jdk-stretch
+
+LABEL MAINTAINER=fx19880617@gmail.com
+
+ARG PINOT_BRANCH=master
+ARG PINOT_GIT_URL="https://github.com/apache/incubator-pinot.git"
+RUN echo "Trying to build Pinot from [ ${PINOT_GIT_URL} ] on branch [ ${PINOT_BRANCH} ]"
+ENV PINOT_HOME=/opt/pinot
+ENV PINOT_BUILD_DIR=/opt/pinot-build
+
+# extra dependency for running launcher
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		vim wget curl git maven\
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN git clone ${PINOT_GIT_URL} ${PINOT_BUILD_DIR}  && \
+    cd ${PINOT_BUILD_DIR} && \
+    git checkout ${PINOT_BRANCH} && \
+    mvn install package -DskipTests -Pbin-dist && \
+    mkdir -p ${PINOT_HOME}/configs && \
+    mkdir -p ${PINOT_HOME}/data && \
+    cp -r pinot-distribution/target/apache-pinot-*-bin/apache-pinot-*-bin/* ${PINOT_HOME}/. && \
+    chmod +x ${PINOT_HOME}/bin/*.sh && \
+    mvn dependency:purge-local-repository -DactTransitively=false -DreResolve=false --fail-at-end && \
+    rm -rf ${PINOT_BUILD_DIR}
+
+VOLUME ["${PINOT_HOME}/configs", "${PINOT_HOME}/data"]
+
+EXPOSE 9000 8099 8098 8097 8096 9514
```

Comment:
What does these numbers stand for? Maybe add some comments?

Reply:
Those are default ports for controller/broker/server/admin.